assets/src/js/query_loop_carousel.js

```diff
@@ -0,0 +1,97 @@
+import {v4 as uuid} from 'uuid';
```

Comment:
The idea for this file is to implement extra functionality which is required by the [carousel](https://getbootstrap.com/docs/4.0/components/carousel/). In addition to that, we need to keep the block template as much simpler as we can.